### PR TITLE
added link to k8s secret video

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: _layouts/index.html [hunk 1]
```diff
@@ -116,6 +116,17 @@ <h3>And much more</h3>
 		<h2 class="text-center top-space">Recent articles about Skydive</h2>
 		<br>
 
+		<div class="row">
+			<div class="col-sm-12">
+				<h3><a href="https://www.youtube.com/watch?v=jl2_jUKspJA">
+					k8s configuration roll-out using Skydive
```
Comment (lebauce) [inline _layouts/index.html]: s/k8s/Kubernetes/

File: _layouts/index.html [hunk 2]
```diff
@@ -116,6 +116,17 @@ <h3>And much more</h3>
...
+				</a></h3>
+				<p>
+In this demo we show how skydive (http://skydive.network) can help roll-out configuration changes across your k8s landscape, helping in visualizing which pods are dependent on the configuration object and going beyond that by automatic the restarting of all dependent pods automatically by utilizing Skydive REST API.
```
Comment (lebauce) [inline _layouts/index.html]: s/skydive/Skydive/

Comment (lebauce) [reply _layouts/index.html]: I don't think we need to put a link to Skydive as we already are on the Skydive website.

File: _layouts/index.html (hunk 1 above)
Comment (hunchback) [reply _layouts/index.html]: ok

File: _layouts/index.html (hunk 2 above)
Comment (hunchback) [reply _layouts/index.html]: ok.

File: _layouts/index.html [hunk 3]
```diff
@@ -131,7 +142,7 @@ <h3><a href="https://www.youtube.com/watch?v=Dwrk03ibces">
 			<div class="col-sm-12">
 				<h3><a href="https://www.youtube.com/watch?v=DOQvtIR6c_A">Troubleshooting Kubernetes applications with Skydive</a></h3>
 				<p>
-					Skydive is a real-time topology and protocol analyzer, with its' k8s plugin it can now be used to debug k8s applications. In this video we demonstrate how it can be used to identify issues within the bookinfo sample application.
+					Skydive is a real-time topology and protocol analyzer, with its' Kubernetes plugin it can now be used to debug k8s applications. In this video we demonstrate how it can be used to identify issues within the bookinfo sample application.
```
Comment (lebauce) [inline _layouts/index.html]: nit: with its' => with it's

Comment (hunchback) [reply _layouts/index.html]: ok

**Pull request merged**